packages/lesswrong/server/codegen/generateDbTypes.ts

```diff
@@ -44,6 +44,8 @@ function databaseTypeToTypescriptType(databaseType: DatabaseBaseType | `${Databa
       return 'any';
     case 'TIMESTAMPTZ':
       return 'Date';
+    default:
+      throw new Error(`Unrecognized database type: ${databaseType}`);
```

Comment:
🤔 falling through to the default case should be ruled out by the type system matching the switch statement on all possible remaining cases in DatabaseBaseType | `${DatabaseBaseType}[]`, after the three type guard above (combined with the return type signature of this function not including `undefined`).  Was something complaining without this?

Reply:
I added it because I did run into this case (but this was only possible because I had ignored a type error elsewhere)